### PR TITLE
Add CreateActCtx and ActivateActCtx

### PR DESCRIPTION
**Pull request by gonutz**
This accompanies my PR for the walk framework.

### CODE REVIEW

File: kernel32.go [hunk 1]
```diff
@@ -79,6 +79,8 @@ var (
 	mulDiv                             uintptr
 	setLastError                       uintptr
 	systemTimeToFileTime               uintptr
+	createActCtx                       uintptr
```
Comment (lxn) [inline kernel32.go]: Please keep these sorted.

File: kernel32.go [hunk 2]
```diff
@@ -143,6 +158,8 @@ func init() {
 	mulDiv = MustGetProcAddress(libkernel32, "MulDiv")
 	setLastError = MustGetProcAddress(libkernel32, "SetLastError")
 	systemTimeToFileTime = MustGetProcAddress(libkernel32, "SystemTimeToFileTime")
+	createActCtx = MustGetProcAddress(libkernel32, "CreateActCtxW")
```
Comment (lxn) [inline kernel32.go]: Please keep these sorted.

File: kernel32.go [hunk 3]
```diff
@@ -349,3 +366,25 @@ func SystemTimeToFileTime(lpSystemTime *SYSTEMTIME, lpFileTime *FILETIME) bool {
 
 	return ret != 0
 }
+
```
Comment (lxn) [inline kernel32.go]: Please keep these sorted.

**Comment by gonutz** [comment]
OK this one is update as well, thanks for doing this and letting me participate, you have created a very nice framework!

Again, not adding my mail address on purpose.

**Comment by lxn** [comment]
Thanks again!

**Pull request merged**